Tests/SwiftParserTest/translated/IfconfigExprTests.swift

```diff
@@ -427,6 +427,28 @@ final class IfconfigExprTests: XCTestCase {
       ]
     )
   }
+  
+  func testIfconfigExpr31() {
+    AssertParse(
+      """
+      #if arch(x86_64)
+        debugPrint("x86_64")
+      #else if arch(arm64)
+        debugPrint("arm64")
+      #else
+        debugPrint("Some other architecture.")
+      #endif
+      """,
+      diagnostics: [
+        DiagnosticSpec(message: """
+        error: unexpected 'if' keyword following '#else' conditional compilation directive; did you mean '#elseif'?
+                            #else if arch(arm64)
+                            ~~~~~~^~
+                            #elseif
+        """)
```

Comment:
The message of the diagnostic doesn’t include the `error: ` prefix, it’s just `unexpected 'if' keyword following '#else' conditional compilation directive; did you mean '#elseif'?`. The fix-its also aren’t part of the message and should be specified in the `fixIts` parameter to `DiagnsoticSpec`. So this should be something like the following (written in GitHub so it might not be 100% correct)

```suggestion
        DiagnosticSpec(message: "unexpected 'if' keyword following '#else' conditional compilation directive; did you mean '#elseif'?", fixIts: ["replace '#else if' by '#elseif"])
```

Reply:
I've included some more `DiagnosticSpec`s because the parser is producing more than the one I previously added (`"unexpected 'if' keyword following '#else' conditional compilation directive; did you mean '#elseif'?"`). I think it's correct.

Is this ok?

Reply:
With the `handledNodes` adjustment you suggested above, the extra diagnostics are gone. I've adjusted the test accordingly.